superset/utils/log.py

```diff
@@ -92,6 +89,13 @@ def log_context(
         if log_to_statsd:
             self.stats_logger.incr(action)
 
+        payload.update(
+            {
+                "path": request.path,
+                "path_no_int": strip_int_from_path(request.path),
```

Comment:
can we name this to `path_no_param` in case we want to update the logic in `strip_int_from_path` as discussed in #11714 ?

Reply:
as mentioned in the summary, this PR:
> Does not:
>
> - Address any naming concerns about the new columns

I'd rather keep this PR just scoped to fixing the migration issue. We can fast follow with naming changes if we all agree they're preferred.

Reply:
I think `path_no_param` was agreed on. Just thought it's better to make that change sooner so we don't have any dirty data.

Reply:
But it you just merge this after CI is agree, I can make another follow PR soon.

Reply:
Feel free to completely drop `path_no_int` and the `strip_int_from_path` method

Reply:
I'm also ok fine with `ref` -> `object_ref` too. Happy to do in a fast follow PR

Reply:
Naming is hard. I'm a fan of measure twice cut once, as renaming things (or dealing with legacy names) in the future is always somewhat painful.

Reply:
ah, i see the `path_no_param` agreement from the previous pr. tbh, CI is sooo slow tonight i probably won't merge this until tomorrow. So i'll make the change now.